src/pathfinder/yapf/yapf_base.hpp

```diff
@@ -218,47 +211,47 @@ class CYapfBaseT {
 	void AddNewNode(Node &n, const TrackFollower &tf)
 	{
 		/* evaluate the node */
-		bool bCached = Yapf().PfNodeCacheFetch(n);
-		if (!bCached) {
-			m_stats_cost_calcs++;
+		bool cached = Yapf().PfNodeCacheFetch(n);
+		if (!cached) {
+			this->stats_cost_calcs++;
 		} else {
-			m_stats_cache_hits++;
+			this->stats_cache_hits++;
 		}
 
-		bool bValid = Yapf().PfCalcCost(n, &tf);
+		bool valid = Yapf().PfCalcCost(n, &tf);
 
-		if (bValid) bValid = Yapf().PfCalcEstimate(n);
+		if (valid) valid = Yapf().PfCalcEstimate(n);
 
 		/* have the cost or estimate callbacks marked this node as invalid? */
-		if (!bValid) return;
+		if (!valid) return;
 
 		/* The new node can be set as the best intermediate node only once we're
 		 * certain it will be finalized by being inserted into the open list. */
-		bool set_intermediate = m_max_search_nodes > 0 && (m_pBestIntermediateNode == nullptr || (m_pBestIntermediateNode->GetCostEstimate() - m_pBestIntermediateNode->GetCost()) > (n.GetCostEstimate() - n.GetCost()));
+		bool set_intermediate = this->max_search_nodes > 0 && (this->best_intermediate_node == nullptr || (this->best_intermediate_node->GetCostEstimate() - this->best_intermediate_node->GetCost()) > (n.GetCostEstimate() - n.GetCost()));
```

Comment:
Excessively long line (not your fault). Could definitely benefit from splitting up.